templates/front/typed-scss-modules.config.ts

```diff
@@ -0,0 +1,4 @@
+export default {
```

Comment:
Il faudrait mettre un commnentaire pour dire : `This is configuration for the ``typed-scss-modules`` package to enable SCSS modules`